### ISSUE TITLE
Add support for protected direct streams, introduced in Recording Service 1.33

### ISSUE THREAD

**Issue by hharsunen** [RayBa82/DVBViewerController#65]
If Recording Service option "Options → Web Settings → Password also for LAN/WLAN" RS option is set and player device IP is not set as "TrustedDevices" in server's svcuserdata.xml then DVBViewerController's open direct stream no longer works in most video players, since the URL requires authentication.

Verified that `http://<username>:<password>@<RS address>:<RS port>/upnp/channelstream/<my channel id>.ts` works. Just like in transcoded streams.

Don't know how important feature this would be but, took me a while to figure that this was the reason why my direct streaming stopped working. If not a property for this, then some info on the preference view where RS address and port is configured that authenticated direct streams are not supported.

My test setup:
RS version: 1.33.2.0
DVBViewerController version: 1.2.0 build 7
```
Settings (local)
RS service URL: http://10.10.10.2
RS port: 8080
Stream url becomes: http://<RS service URL>:<RS port>/upnp/channelstream/<my channel id>.ts #doesn't work if "Password also for LAN/WLAN" is set
This would work: http://<username>:<password>@<RS address>:<RS port>/upnp/channelstream/<my channel id>.ts #this works in both scenarios if "Password also for LAN/WLAN" is set or not set

Settings (remote)
RS service URL: https://<my RS reverse proxy public hostname>
RS port: 443
Stream url becomes: https://<username>:<password>@<RS service URL>/flashstream/stream.ts?preset=TS%20Mid%201200%20kbit&ffPreset=ultrafast&chid=<my chnnel id> #this works
```

**Comment by hharsunen** [comment]
Or just try the url internally within app and if there's 401, then add the username+password in the URL to be forwarded to external player

**Comment by RayBa82** [comment]
fixed since 1.2.0-10
